### PR TITLE
Add example using helm repo to helm installation guide

### PR DESCRIPTION
**Pull request by avrittrohwer**
#### What type of PR is this?

/kind documentation

#### What this PR does / why we need it:

Makes the helm OCI repo more discoverable.

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

### CODE REVIEW

File: charts/kueue/README.md [hunk 1]
```diff
@@ -24,12 +24,21 @@ Quick start instructions for the setup and configuration of kueue using Helm.
...
 $ git clone git@github.com:kubernetes-sigs/kueue.git
 $ cd kueue/charts
 $ helm install kueue kueue/ --create-namespace --namespace kueue-system
 ```
 
+Or use the charts pushed to `us-central1-docker.pkg.dev/k8s-staging-images/charts/kueue`:
+
+```bash
+VERSION=v0.9.0
+helm install kueue oci://us-central1-docker.pkg.dev/k8s-staging-images/charts/kueue --version="${VERSION}" --create-namespace --namespace=kueue-system
```
Comment (mbobrovskyi) [inline charts/kueue/README.md:39]:
```suggestion
helm install kueue oci://us-central1-docker.pkg.dev/k8s-staging-images/charts/kueue --version="v0.9.0" --create-namespace --namespace=kueue-system
```

File: charts/kueue/README.md [hunk 2]
```diff
@@ -24,12 +24,21 @@ Quick start instructions for the setup and configuration of kueue using Helm.
...
 ```bash
 $ git clone git@github.com:kubernetes-sigs/kueue.git
 $ cd kueue/charts
 $ helm install kueue kueue/ --create-namespace --namespace kueue-system
 ```
 
+Or use the charts pushed to `us-central1-docker.pkg.dev/k8s-staging-images/charts/kueue`:
+
+```bash
+VERSION=v0.9.0
```
Comment (mbobrovskyi) [inline charts/kueue/README.md:38]: Could you please add file path for this file here

https://github.com/kubernetes-sigs/kueue/blob/43b7374764498604e447c312b36bae521c472f42/Makefile#L282

to replace version on `make prepare-release-branch`.

Comment (avrittrohwer) [reply charts/kueue/README.md]: How about https://github.com/kubernetes-sigs/kueue/pull/3517/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R283?

Comment (avrittrohwer) [reply charts/kueue/README.md]: Tested with `$ sed -r 's/--version="v[0-9]+\.[0-9]+\.[0-9]+/--version="v0.9.1/g' -i charts/kueue/README.md`

```
$ gd
diff --git a/Makefile b/Makefile
index dcf8edbb..0e343dd8 100644
--- a/Makefile
+++ b/Makefile
@@ -280,6 +280,7 @@ artifacts: kustomize yq helm ## Generate release artifacts.
 .PHONY: prepare-release-branch
 prepare-release-branch: yq kustomize ## Prepare the release branch with the release version.
        $(SED) -r 's/v[0-9]+\.[0-9]+\.[0-9]+/$(RELEASE_VERSION)/g' -i README.md -i site/hugo.toml -i charts/kueue/README.md
+       $(SED) -r 's/--version="v[0-9]+\.[0-9]+\.[0-9]+/--version="$(RELEASE_VERSION)/g' -i charts/kueue/README.md
        $(YQ) e '.appVersion = "$(RELEASE_VERSION)"' -i charts/kueue/Chart.yaml
        @$(call clean-manifests)

diff --git a/charts/kueue/README.md b/charts/kueue/README.md
index fbcb94d9..6609cec9 100644
--- a/charts/kueue/README.md
+++ b/charts/kueue/README.md
@@ -35,7 +35,7 @@ $ helm install kueue kueue/ --create-namespace --namespace kueue-system
 Or use the charts pushed to `us-central1-docker.pkg.dev/k8s-staging-images/charts/kueue`:

 ```bash
-helm install kueue oci://us-central1-docker.pkg.dev/k8s-staging-images/charts/kueue --version="v0.9.0" --create-namespace --namespace=kueue-system
+helm install kueue oci://us-central1-docker.pkg.dev/k8s-staging-images/charts/kueue --version="v0.9.1" --create-namespace --namespace=kueue-system
 ```
```

Comment (mbobrovskyi) [reply charts/kueue/README.md]: Thanks!

File: Makefile [hunk 3]
```diff
@@ -279,7 +279,8 @@ artifacts: kustomize yq helm ## Generate release artifacts.
 
 .PHONY: prepare-release-branch
 prepare-release-branch: yq kustomize ## Prepare the release branch with the release version.
-	$(SED) -r 's/v[0-9]+\.[0-9]+\.[0-9]+/$(RELEASE_VERSION)/g' -i README.md -i site/hugo.toml
+	$(SED) -r 's/v[0-9]+\.[0-9]+\.[0-9]+/$(RELEASE_VERSION)/g' -i README.md -i site/hugo.toml -i charts/kueue/README.md
```
Comment (mbobrovskyi) [inline Makefile:282]:
```suggestion
	$(SED) -r 's/v[0-9]+\.[0-9]+\.[0-9]+/$(RELEASE_VERSION)/g' -i README.md -i site/hugo.toml
```

Only one thing. Could you please revert?

Comment (avrittrohwer) [reply Makefile]: Ah, good catch!

**Comment by mbobrovskyi** [comment]
/lgtm
Thanks!

**Review by mimowo — COMMENTED** [review]
This is awesome, thank you!
/lgtm
/approve
/cherry-pick website

**Comment by mbobrovskyi** [comment]
Do we need to cherry-pick to website? Looks like it just a helm docs changes.

**Comment by mimowo** [comment]
Helm docs are part of documentation, so it feels better to cherry-pick it.

**Comment by mimowo** [comment]
Ah, I see, we reference the link from "main" for helm, so it already displays the information for those who follow the link.
So no cherry-pick needed. We may though consider the commands to be part of website before following the link, but this is for another (potential) improvement. Thanks for the check @mbobrovskyi

**Pull request merged**